docs/configuration.md

```diff
@@ -28,7 +28,7 @@ hostname = '0.0.0.0'
 port = 80
 ```
 
-## HTTPS
```

Comment:
Maybe "HTTPS and certificates" in the title?

Then we can probably explain a bit more in the following text, or maybe a subsection. But I think we need to guide a bit more the user about it, especially when you search for it :)

What do you think @julien-f ?